### PR TITLE
Pivotal Tracker Workflow at Ombu Labs

### PR DESCRIPTION
**Pull request by etagwerker**
Hey @cleicar, 

I added a draft for an article about Pivotal Tracker and our workflow. 

Please check it out and let me know if you have any feedback.

Thanks!

### CODE REVIEW

File: _posts/2018-08-20-how-we-use-pivotal-tracker-at-ombu-labs.markdown [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+There are many things that are easier with Pivotal Tracker, as long as you are
+using it the right way. Some of its features are very useful for agile teams like
+ours. This is how we like to use it to ship value with every sprint and keep
+track of our [velocity](http://wiki.c2.com/?ProjectVelocity).
+
+<!--more-->
+
+## Step 1: Filing user stories
+
+Every thing has to be tracked according to the type of story. Most of the user
```
Comment (mauro-oto) [inline _posts/2018-08-20-how-we-use-pivotal-tracker-at-ombu-labs.markdown]: It's weird seeing `Every thing has to be` separate, I think `Everything has to be` is more natural.

File: _posts/2018-08-20-how-we-use-pivotal-tracker-at-ombu-labs.markdown [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+sequence to estimate stories.
+
+If a story is more complicated than 8 points, you should break it down in two or
+more stories. When you estimate, you must remember that you are [estimating
+complexity](https://rubygarage.org/blog/3-reasons-to-estimate-with-story-points),
+**not hours**. Different developers might take different amount of time
+in completing that story, so **we shouldn't estimate hours**.
+
+For the example we mentioned before, we estimate that that story has 3 points of
+complexity.
```
Comment (mauro-oto) [inline _posts/2018-08-20-how-we-use-pivotal-tracker-at-ombu-labs.markdown]: I think this is something we've discussed in the past, but shouldn't there be a points to hours mapping? Or does that come after the team has worked on one or more sprints, and you can tell from the velocity and # of hours worked? 

I get the idea of using points to estimate features/bugs, but in certain scenarios (when we estimate Rails Upgrades or MVPs) we use hours, not points.

**Comment by lubc** [comment]
Cool! Wonder if it worth adding slack integration to our pivotal tracker projects. If we do that we can mention it in the article.

File: _site/page4/index.html [hunk 3]
```diff
@@ -285,7 +310,7 @@ <h2 class="post-title">
       <a href="/blog/tags/backbone"
          title="Articles about Backbone">Backbone</a>,
       <a href="/blog/tags/security"
-         title="Articles about Security">Security</a><time class="post-date" datetime="2015-12-15">
+         title="Articles about Security">Security</a><time class="post-date" datetime="2015-12-16">
```
Comment (mauro-oto) [inline _site/page4/index.html]: @etagwerker there's a number of unrelated modifications related to timezones, you probably regenerated the blog using Ruby 2.4 or 2.5. 

I have to regenerate it using Ruby 2.3, otherwise it doesn't respect the timezone set in `config.yml` for some reason. This is fixed by #144.

File: _posts/2018-08-20-how-we-use-pivotal-tracker-at-ombu-labs.markdown (hunk 1 above)
Comment (mauro-oto) [reply _posts/2018-08-20-how-we-use-pivotal-tracker-at-ombu-labs.markdown]: It'd make sense if it said `Every single thing`.

File: _posts/2018-08-20-how-we-use-pivotal-tracker-at-ombu-labs.markdown (hunk 2 above)
Comment (etagwerker) [reply _posts/2018-08-20-how-we-use-pivotal-tracker-at-ombu-labs.markdown]: We should always use points. I'll add a paragraph to explain a bit more about that.

I believe points give us more flexibility in terms of calculating when a story will be done.

File: _site/page4/index.html (hunk 3 above)
Comment (etagwerker) [reply _site/page4/index.html]: @mauro-oto ah, weird! I just regenerated it with Ruby 2.3.1

File: _posts/2018-08-20-how-we-use-pivotal-tracker-at-ombu-labs.markdown [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+title:  "How we use Pivotal Tracker at Ombu Labs"
+date: 2018-08-20 11:10:00
+categories: ["agile", "pivotal-tracker"]
+author: "etagwerker"
+---
+
+We tend to use [GitHub](https://github.com) for everything at [Ombu Labs](https://www.ombulabs.com), but sometimes our clients
+prefer [Pivotal Tracker](https://www.pivotaltracker.com). We have no problem
+with Pivotal Tracker, we just like to keep everything in one place, that is why
+we prefer GitHub for everything.
```
[the remark this answers is not in the document]
Comment (etagwerker) [reply _posts/2018-08-20-how-we-use-pivotal-tracker-at-ombu-labs.markdown]: I rephrased the entire conclusion. Please check it out!

**Comment by etagwerker** [comment]
@cleicar @mauro-oto @lubc Just addressed your comments. Also, I expanded the conclusion. 

Please check it out and let me know!

**Comment by etagwerker** [comment]
> Cool! Wonder if it worth adding slack integration to our pivotal tracker projects. If we do that we can mention it in the article.

I recently added this integration. I don't know it is a great idea. I believe we will need to tweak the integration so that it is less noisy. For now I won't mention it in this article.
